.editorconfig

```diff
@@ -7,3 +7,7 @@ indent_style = tab
 indent_size = 4
 charset = utf-8
 trim_trailing_whitespace = true
+
+[*.scss]
+indent_style = space
```

Comment:
Keep tabs, but let indent_size be 2.

This should also be reflected in Prettierrc

Reply:
I removed the line 12 on this PR and added tabWidth in prettierrc

Reply:
Thanks!